### PR TITLE
Tweak: Usage - Include non elementor pages [ED-3023]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix the conflict

File: schemas/usage.json [hunk 1]
```diff
@@ -562,37 +588,20 @@
...
-								}
-							},
-							"additionalProperties": false
-						}
-					},
-					"additionalProperties": false
+					"$ref": "#/definitions/posts_per_type_per_post_status",
+					"title": "The posts schema"
+				},
+				"non-elementor-posts": {
```
Comment (matipojo) [inline schemas/usage.json:601]: Please add an example

**Review by matipojo — CHANGES_REQUESTED** [review]

File: schemas/usage.json [hunk 2]
```diff
@@ -562,37 +588,20 @@
 			"description": "The usages of various components in Elementor.",
 			"required": [
 				"posts",
+				"non-elementor-posts",
```
Comment (matipojo) [inline schemas/usage.json:591]: Please add a "Change Log" section ( on a `md` file?) with a line "3.3.0 - added `usage/non-elementor-post` for count posts that not built with Elementor.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: schemas/usage.json (hunk 1 above)
Comment (iNewLegend) [reply schemas/usage.json]: Handled.

File: schemas/usage.json (hunk 2 above)
Comment (iNewLegend) [reply schemas/usage.json]: Handled.

File: schemas/usage.json (hunk 1 above)
Comment (iNewLegend) [reply schemas/usage.json]: @matipojo maybe "non-edit-with-elementor-posts"?

**Review by matipojo — APPROVED** [review]

**Pull request merged**